docs/installation.rst

```diff
@@ -186,7 +186,7 @@ Install the dependencies with development headers::
     liblz4-dev libzstd-dev \
     build-essential \
     pkg-config python3-pkgconfig
-    sudo apt-get install libfuse-dev fuse    # optional, for FUSE support
+    sudo apt-get install libfuse-dev fuse python3-llfuse # optional, for FUSE support
```

Comment:
I don't think this is a valid change. But you have likely found some other issue.

In the docs we already have:
```
some other Python dependencies, pip will automatically install them for you.
```
That includes llfuse and this is the reason why your change is not valid.

But, we also have that:
```
optionally, the llfuse_ Python package is required if you wish to mount an archive as a FUSE filesystem. See setup.py about the version requirements.
```
This is a bit superfluous (as that is already included in what I mentioned above) and also maybe misleading.

Reply:
> I don't think this is a valid change. But you have likely found some other issue.
> 
> In the docs we already have:
> 
> ```
> some other Python dependencies, pip will automatically install them for you.
> ```
> 
> That includes llfuse and this is the reason why your change is not valid.
> 
> But, we also have that:
> 
> ```
> optionally, the llfuse_ Python package is required if you wish to mount an archive as a FUSE filesystem. See setup.py about the version requirements.
> ```
> 
> This is a bit superfluous (as that is already included in what I mentioned above) and also maybe misleading.

Thomas, in my systems, that dependency was _not_ installed - I followed your documentation. You are working with borg since years - I try to install on fresh machines. (here: Linux Mint 19.3, many instances, was lacking that dependency.

Arguendo you were right with your Behauptungen, the llfuse was missing nevertheless.  I followed strictly the installation rules in the borg documentation.

Reply:
from stable docs:
```
# install Borg + Python dependencies into virtualenv
pip install borgbackup
# or alternatively (if you want FUSE support):
pip install borgbackup[fuse]                          <------- that installs llfuse
```

Reply:
and the reason why this is not done by default is that fuse support is not present / unproblematic on all platforms.